skopt/utils.py

```diff
@@ -44,3 +52,58 @@ def create_result(Xi, yi, space=None, rng=None, specs=None, models=None):
     res.random_state = rng
     res.specs = specs
     return res
+
+
+def dump(res, filename):
+    """
+    Store an skopt optimization result into a file.
+
+    If the objective function (`.specs['args']['func']`) is unserializable,
+    it will be removed before saving into file.
+
+    Parameters
+    ----------
+    * `res` [`OptimizeResult`, scipy object]:
+        Optimization result object to be stored.
+
+    * `filename` [string]:
+        The path of the file in which it is to be stored.
+    """
+
+    try:
+        with open(filename, 'wb') as f:
+            pickle.dump(res, f)
```

Comment:
It was discussed in #190, somewhere around [this comment](https://github.com/scikit-optimize/scikit-optimize/issues/190#issuecomment-249296787)

I'm ok with `joblib`.